cmd/oa.js

```diff
@@ -10,7 +8,7 @@ if( process.argv.length !== 4 ){
   process.exit(1);
 }
 
-if( tty.isatty( process.stdin ) ){
+if( process.stdin.isTTY ){
```

Comment:
Cool, didn't know you could do this! It removes a require line as well :+1:

Reply:
I don't think it used to be possible in older versions of node...